packages/design/src/Borders.mdx

```diff
@@ -0,0 +1,64 @@
+---
+name: Dividing Content
```

Comment:
I understand that this name is probably more appropriate as we are also talking about `Divider` here. However it did take me a few seconds to figure out where to go to look for border information.

Reply:
That would explain why I couldn't find it in the docs. 😂